gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -120,6 +120,7 @@ def print_help(self):
 >   ov          overview of the cryptocurrencies,       e.g.: market cap, DeFi, latest news, top exchanges, stables{dim}
 >   dd          due-diligence for loaded coin,          e.g.: coin information, social media, market stats
 >   ta          technical analysis for loaded coin,     e.g.: ema, macd, rsi, adx, bbands, obv
+>   pred        prediction techniques
```

Comment:
Maybe it's good idea to add there some examples like in commands above. What do you think ?

Reply:
Oops.  Yeah was being lazy.